src/Model/Thread.php

```diff
@@ -30,6 +32,12 @@ class Thread
     private $content;
 
     /**
+     * @OneToMany(targetEntity="Comment", mappedBy="thread", fetch="EXTRA_LAZY")
```

Comment:
what does `fetch="EXTRA_LAZY"` do?

Reply:
http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/tutorials/extra-lazy-associations.html
When counting the threads, I don't need the all the comment entities to be loaded, and I figured thought this could be helpful